code/game/mecha/mecha.dm

```diff
@@ -840,6 +840,9 @@
 	if(repairing)
 		to_chat(user, "<span class='notice'>[src] is currently being repaired!</span>")
 		return
+	if(state < 1)
```

Comment:
```suggestion
	if(state == 0)
```
The state numbers are just arbitrary unordered magic numbers. It does not make sense to compare them for greater than/less than, just as it doesn't make sense to compare `"normal mode"` `"maintenance mode"` `"power cell removed"` etc. via `<`.